Application/src/visuals/Message.java

```diff
@@ -8,7 +8,8 @@
 /**
  * Class for creating the messages to pass between "classes".
  * @author Sebastian Fransson
- * @version 1.0
+ * -collaborator Rashad Kamsheh
+ * @version 1.1
```

Comment:
Version 2.0?

Reply:
@IsabelleTornqvist 
1.0++ is 1.1
1.9++ is 2.0